### PR TITLE
Video landing page, index.html, header

### PR DESCRIPTION
**Pull request by ltagliaferri**
- `index.html`: I added Learning Labs link to the home page (replacing Wolfi), and updated Tag History API doc to be the Diff API doc.
- `script-header.html`: Request from Marketing (probably could stuff in a variable like with GA if it's preferred)
- Images landing page: I swapped out Ruby for NGINX
- Videos landing page: Now that we have a critical mass, I made this look nicer ✨

### CODE REVIEW

**Comment by amouat** [comment]
LGTM, couple of small things:

 - Where is the learning labs link meant to appear?
 - How do we get to this page?

**Review by amouat — APPROVED** [review]

File: layouts/index.html [hunk 1]
```diff
@@ -210,7 +210,7 @@ <h4 class="section-title mb-2">Reach 'inbox zero' for your CVEs</h4>
                 </div>
                 <div class="d-flex flex-column">
                   <div class="caption-title mb-2">
-                    Tag History API
+                    Diff API
                   </div>
                   <div class="caption">How to use the Chainguard Images Tag History API to fetch the tag history of image variants
```
Comment (erikaheidi) [inline layouts/index.html:215]:
```suggestion
                  <div class="caption">How to use the Chainguard Images Diff API to compare changes between image builds
```

**Review by erikaheidi — CHANGES_REQUESTED** [review]
Everything looks good, just the description of the Diff API was still referring to the Tag history api

**Comment by ltagliaferri** [comment]
> LGTM, couple of small things:
> 
> * Where is the learning labs link meant to appear?
> * How do we get to this page?

The gradient image and text for the learning labs section is all clickable and will take you to the link.

The Videos page is available if you click on "Videos" in the menu bar

**Pull request merged**